blocks/rapid-result/rapid-result.js

```diff
@@ -0,0 +1,139 @@
+import { readBlockConfig, fetchPlaceholders, decorateIcons } from '../../scripts/aem.js';
+import { createElement, observe } from '../../scripts/blocks-utils.js';
+
+// TODO: This is dummy function that fetch sample data from EDS json.
+// It will be replaced when API call is available.
+async function fetchRapidResultMockData() {
+  let hostUrl = window.location.origin;
+  if (!hostUrl || hostUrl === 'null') {
+    // eslint-disable-next-line prefer-destructuring
+    hostUrl = window.location.ancestorOrigins[0];
+  }
+  const apiUrl = `${hostUrl}/draft/jiang/rapidresult.json`;
```

Comment:
you can create a json file in moc api similar to https://github.com/aemsites/icicidirect/blob/main/scripts/mock-blogdata.json